app/event/send-payment-request-invalid-events.js

```diff
@@ -28,4 +40,23 @@ const sendPaymentRequestInvalidEvents = async (paymentRequests) => {
   }
 }
 
+const sendV2PaymentRequestInvalidEvents = async (paymentRequests) => {
+  if (paymentRequests?.length) {
+    const events = paymentRequests.map(createEvent)
```

Comment:
Do we need any validation on the object as on line 23/23 above?

Reply:
The original validation isn't really adding any value.  It's just checking that the value is an object which isn't really helpful.

With V2 events, the message is validated within the npm package so we don't need to duplicate that in each app.